cirq/sim/mux.py

```diff
@@ -23,11 +23,19 @@
 
 from cirq import circuits, protocols, study, devices, ops, value
 from cirq.sim import sparse_simulator, density_matrix_simulator
+from cirq.sim.clifford import clifford_simulator
 
 if TYPE_CHECKING:
     import cirq
 
 
+def _is_clifford_circuit(program: 'cirq.Circuit') -> bool:
+    return all(
```

Comment:
Don't treat measurement differently from the other operations.

Reply:
Done

Reply:
Had to treat them differently since I can't directly add them to the List of gates. Should I also limit to terminal measurements here for now?